lib/harmony/harmonydb/sql/20230712.sql

```diff
@@ -0,0 +1,37 @@
+create table SectorLocation
+(
+    "miner_id"        int8,
+    "sector_num"   int8,
+    "sector_filetype" int,
+    "storage_id"      varchar,
+    "is_primary"      bool,
+    constraint SectorLocation_pk
+        primary key ("miner_id", "sector_num", "sector_filetype", "storage_id")
+);
+
+
+create table StorageLocation
+(
+    "storage_id"  varchar not null
+        constraint "StorageLocation_pkey"
+            primary key,
+    "urls"       varchar,
+    "weight"     int8,
+    "max_storage" int8,
+    "can_seal"    bool,
+    "can_store"   bool,
+    "groups"     varchar,
+    "allow_to"    varchar,
```

Comment:
Wouldn't making those into [arrays](https://www.postgresql.org/docs/current/arrays.html) make some queries easier?

Reply:
I originally defined these as arrays but that seemed to cause more problems than it solved. Sticking to more fundamental types seems to be the better course of action